### PR TITLE
Implement global (controller) filter for checking

### PR DESCRIPTION
**Pull request by HlibBondarev**
Implement Action Filter for controller

### CODE REVIEW

File: Streetcode/Streetcode.BLL/ActionFilters/AsyncValidateEntityExistsFilter.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        if (id != INEXISTEDID)
+        {
+            var entity = await _repositoryBase.GetFirstOrDefaultAsync(e => e.Id.Equals(id));
+
+            if (entity == null)
+            {
+                string errorMessageType = GetErrorMessage(context.HttpContext.Request.Method);
+                var errorMsg = string.Format(errorMessageType, typeof(T).Name, id);
+                var expectedMessage = string.Format(
```
Comment (yaninavoda) [inline Streetcode/Streetcode.BLL/ActionFilters/AsyncValidateEntityExistsFilter.cs:48]: I can't see where expectedMessage is used

File: Streetcode/Streetcode.BLL/ActionFilters/AsyncValidateEntityExistsFilter.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            {
+                context.HttpContext.Items.Add("entity", entity);
+            }
+        }
+
+        var result = await next();
+    }
+
+    private string GetErrorMessage(string requestType) => requestType switch
+    {
```
Comment (yaninavoda) [inline Streetcode/Streetcode.BLL/ActionFilters/AsyncValidateEntityExistsFilter.cs:66]: Make GetErrorMessage static

**Review by yaninavoda — COMMENTED** [review]
Added 2 comment in AsyncValidateEntityExistsFilter

**Comment by HlibBondarev** [comment]
fixed everything

**Review by yaninavoda — APPROVED** [review]

**Review by OlhaPivniuk — APPROVED** [review]

**Pull request merged**